demisto_sdk/commands/unify/integration_script_unifier.py

```diff
@@ -324,35 +322,35 @@ def check_api_module_imports(script_code: str) -> Tuple[str, str]:
         # General regex to find API module imports, for example: "from MicrosoftApiModule import *  # noqa: E402"
         module_regex = r'from ([\w\d]+ApiModule) import \*(?:  # noqa: E402)?'
 
-        module_match = re.search(module_regex, script_code)
-        if module_match:
-            return module_match.group(), module_match.group(1)
+        module_matches = re.finditer(module_regex, script_code)
 
-        return '', ''
+        return {module_match.group(): module_match.group(1) for module_match in module_matches}
```

Comment:
does it cover cases where there is no `group(1)`?

Reply:
group(0) means the full match and group(1) is the first capturing group. It will always be there